website/content/docs/nia/configuration.mdx

```diff
@@ -206,9 +207,15 @@ task {
 - `regexp` - (string) Optional if [`task.services`](/docs/nia/configuration#services) is configured. Either `regexp` or `task.services` or both must be configured. Only services that have a name which matches the regular expression are used by the task. If not provided, `regexp` will default to an exact match on `task.services` list. For example, if `task.services = ["api", "web"]`, then `regexp` will default to `^api$|^web$`. See [Task Execution: Catalog Services Condition](/docs/nia/tasks#catalog-services-condition) for more details on the relationship between `task.services` and `regexp`. Some resources for more information on regular expressions: [regular expression syntax](https://github.com/google/re2/wiki/Syntax), [try out regular expression string matching](https://golang.org/pkg/regexp/#Regexp.MatchString).
 - `source_includes_var` - (bool: false) Whether or not the module configured at [`task.source`](#source) includes the [`catalog_services` variable](/docs/nia/terraform-modules#catalog-services-variable). Please refer to the documentation of the selected module for guidance on how to configure this field. If configured inconsistently with the module, Consul-Terraform-Sync will error and exit.
 
+# Network Drivers
```

Comment:
The drop-down only displays for `##` headers and that would cause the two drivers to disappear. I'm ok with `# Network Drivers` not showing, it's more of a handy anchor for the URL https://consul.io/docs/nia/configuration#network-drivers

What do you think?

Reply:
🤔 my personal preference is for `Network Drivers` to be in the jump section since it seems "more at the level" with the other sections listed in the drop down and will be more consistent formatting-wise - not sure if I described that well.

It's not a strong preference by any means. I don't mind if we think that the specific driver links are more useful to jump to, which is more important. I mostly wanted to double-check your intention. I'm ok with you keeping it as is!

Reply:
It was intentional for me, but you're right that the `## Network Drivers` would be the most consistent way to handle the section tiering.

My inclination is that the individual drivers are more useful to jump to. The `## Terraform Driver` section is long and takes up the whole page and would require someone to click "Jump to" -> "Network Drivers" -> scroll to get to Terraform Cloud. 

Having both of them listed would increase visibility of the new driver. Another option could be to have all 3 of them as `##` to be listed 😀

Reply:
Agree that the individual drivers are more useful and the scrolling around is cumbersome. I like your idea of all of them being `##` !